embedded-kafka/src/main/java/com/playtika/test/kafka/properties/KafkaConfigurationProperties.java

```diff
@@ -63,25 +63,6 @@ public KafkaConfigurationProperties() {
         this.setCapabilities(Arrays.asList(Capability.NET_ADMIN));
     }
 
-    /**
```

Comment:
Host's port to which kafka container server port is exposed are not same since #387: withFixedExposedPort statement was dropped in that commit so host's mapped port is chosen dynamically by docker. Ports specified in KafkaConfigurationProperties are for kafka server only now. So there are no obvious reasons to choose them from set of available hosts' ports.